scripts/data_prep/convert_text_to_mds.py

```diff
@@ -0,0 +1,352 @@
+# Copyright 2022 MosaicML LLM Foundry authors
+# SPDX-License-Identifier: Apache-2.0
+
+import copy
+import math
+import os
+import tempfile
+from argparse import ArgumentParser, Namespace
+from concurrent.futures import ProcessPoolExecutor
+from glob import glob
+from typing import Iterable, List, Tuple, cast
+
+from composer.utils import (ObjectStore, maybe_create_object_store_from_uri,
+                            parse_uri)
+from streaming import MDSWriter
+from tqdm import tqdm
+from transformers import AutoTokenizer
+
+from llmfoundry.data import ConcatTokensDataset
+from llmfoundry.utils.data_prep_utils import (DownloadingIterable,
+                                              merge_shard_groups)
+
+
+def parse_args() -> Namespace:
+    """Parse commandline arguments."""
+    parser = ArgumentParser(
+        description=
+        'Convert text files into MDS format, optionally concatenating and tokenizing'
```

Comment:
tip: add a trailing comma at the end so that lint re-structure the code uniformly. For example
```suggestion
        'Convert text files into MDS format, optionally concatenating and tokenizing',
```

Please do the same for other argparse as well.